spec/models/document_type/body_field_spec.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
This could be fixed up to the commit where body_field_spec is introduced.

Reply:
Good spot - fixed.